lib/commands/web.js

```diff
@@ -433,12 +437,20 @@ helpers.parseExecuteResponse = function (res) {
 };
 
 helpers.parseElementResponse = function (element) {
-  let objId = element.ELEMENT;
+  let objId = helpers.getElementId(element);
   let clientId = (ELEMENT_OFFSET + this.webElementIds.length).toString();
   this.webElementIds.push(objId);
   return {ELEMENT: clientId};
 };
 
+helpers.getElementId = function (element) {
```

Comment:
this looks good but is it the same as unwrapElement?

Reply:
It is yeah. I'll update that.